packages/edit-site/src/components/global-styles/screen-revisions/index.js

```diff
@@ -138,19 +138,21 @@ function ScreenRevisions() {
 			{ hasRevisions &&
 				( editorCanvasContainerView ===
 				'global-styles-revisions:style-book' ? (
-					<StyleBook
+					<GlobalStylesStyleBook
```

Comment:
As explanation: a shared component is required as revisions needs to send a global styles config to the style book.